### PR TITLE
A more elaborate Contributing page

### PR DESCRIPTION
**Pull request by simonv3**
I basically took some of the ideas of the thrown around here https://github.com/opensourcedesign/organization/issues/54 

and here https://github.com/opensourcedesign/organization/issues/53

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,15 +1,50 @@
...
 
+Welcome! 
+
+Before diving into Open Source Design, have a look at our mission: 
+
+> We are a **community** of **designers and developers** pushing more open design processes and improving the design of **open source** apps.
+
+There are a couple of important words there to keep in mind before exploring Open Source Design. 
+
+* Community: Open Source Design is first and foremost a community of people. We hang out in IRC, we chat in our forum, and we work on projects together on GitHub. There is a very loose hierarchy, but it's largely a based on how long you've been involved in the group. People who have been here longer know how the community works and how it operates. If you want to learn more about the rules we have formulated over time, check out our [by-laws](http://opensourcedesign.net/by-laws/) and our [code of conduct](opensourcedesign.net/by-laws/).
```
Comment (RichardLitt) [inline CONTRIBUTING.md]: By laws and code go to the same place? The second link is also missing the protocol.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,15 +1,50 @@
...
+
+Have a look at our [job board](opensourcedesign.net/jobs/).
+
+## Looking for a designer?
+
+Post something to our [job board](opensourcedesign.net/jobs/job-form/).
+
+## Want to talk about design? 
+
+Join us on our forum and [tell us about yourself](https://discourse.opensourcedesign.net/t/introductions-come-in-say-hi/41).
```
Comment (RichardLitt) [inline CONTRIBUTING.md]: I didn't even know there was a Discourse. :/

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,15 +1,50 @@
...
+
+Good! A whole bunch of us organize events all over the world. Get involved here on our [Event Organizing](https://github.com/opensourcedesign/events) repo.
+
+## Hope to get involved in making Open Source Design run?
+
+Have a look at the [Organization Repo](https://github.com/opensourcedesign/organization)
+
+## Want to fund Open Source Design projects? (YES)
+
+[Give us some spare change](opencollective.com/opensourcedesign) every month and we will make cool stuff happen.
```
Comment (RichardLitt) [inline CONTRIBUTING.md]: Needs protocol.

**Comment by simonv3** [comment]
Thanks for the once-over @RichardLitt! Fixed your suggestions.

**Comment by RichardLitt** [comment]
LGTM.

**Pull request merged**